deploy/contracts/docs/contract-architecture.md

```
@@ -0,0 +1,990 @@
+├── contract.rs         # メインコントラクト実装
+├── msg.rs              # メッセージ定義
+├── state.rs            # ストレージ定義
+├── ao_integration.rs   # AO Network プロセス間通信統合
+├── tests.rs            # 基本テスト
+├── owner/              # Owner-Process モジュール
+│   ├── mod.rs          # モジュール統合とFacade
+│   ├── handlers.rs     # メッセージハンドラ
+│   └── storage.rs      # ストレージ操作
+├── holder/             # Holder-Process モジュール
+│   ├── mod.rs          # モジュール統合とFacade
+│   ├── handlers.rs     # メッセージハンドラ
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)